pkg/scheduler/factory.go

```diff
@@ -237,7 +235,7 @@ func (c *Configurator) createFromConfig(policy schedulerapi.Policy) (*Scheduler,
 				}
 			}
 		}
-		args.NodeResourcesFitArgs = &noderesources.FitArgs{
+		args.NodeResourcesFitArgs = &schedulerapi.NodeResourcesFitArgs{
```

Comment:
uhm.... I think we will have to use v1alpha2 types here too. But I would leave this to api reviewers.